roles/restore/tasks/deploy_awx.yml

```diff
@@ -8,9 +8,32 @@
       bash -c "cat '{{ backup_dir }}/awx_object'"
   register: awx_object
 
-- name: Set AWX spec variable from backup
+- name: Create temp file for spec dict
+  tempfile:
+    state: file
+  register: tmp_spec
+
+- name: Write spec vars to temp file
+  shell: |
+    echo '{{ awx_object.stdout }} > {{ tmp_spec.path }}'
+  changed_when: true
```

Comment:
This failed the linter ([here](https://github.com/ansible/awx-operator/pull/279/checks?check_run_id=2501146267)).  Because this only writes to a tempfile which will be cleaned up when the process running the ansible playbook ends, I think this is OK.  If there is a better way to do this, please let me know.

Reply:
I was able to remove this changed_when flag and make this a copy task.